db_assessment/import_db_assessment.py

```diff
@@ -662,4 +690,103 @@ def insertErrors(invalidfiles,op_df,gcpProjectName,bq_dataset):
         print ('\nWARNING: Issues while pusing Errors into operrors table with error ', pushErr)
 
 
+def populateBT(tableName,df,dataframeornot,invalidfiles,btsource,rowsimported,importresults):
+    # Fuction to populate the importresults list which will be used to print using Beautiful Table 
+    # rowsimported of <0 is used to indicate a FAILED status 
+    # fileName='dummy'
+    tmpdataFrame=pd.DataFrame()
+
+    if btsource=='fromimportDataframeToBQ':
+
+        if dataframeornot is not None:
+            if 'PKEY' in df.columns.to_list():
+                # df2.reset_index(inplace=True)
+                if 'dbsizing_summary' not in tableName: # Need to add code to support dbsizing_summary or fix the issue with it 
+                    pkeygroupby=df.groupby(['PKEY']).size()
+                    # pkeygroupby=df2.groupby(level=0).size()
+                    pkeycount=pkeygroupby.to_dict()
+                    for pkeyname,pkeyname_rowcount in pkeycount.items():
```

Comment:
Can we work around the hardcoded stuff?

Reply:
dbsizing_summary + Elapsed

Reply:
hardcoded 'dbsizing_summary' is removed in latest commit for bug132 branch